components/create-collaboration/screen-three.tsx

```
@@ -1,38 +1,436 @@
+                    }}
+                  />
+                  Add Link
+                </Button>
+              }
+              theme={theme}
+              title="External Links"
+              titleStyle={{
+                fontSize: 16,
+              }}
+            >
+              <></>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)